packages/flutter/lib/src/material/pickers/date_picker_dialog.dart

```diff
@@ -93,6 +93,7 @@ Future<DateTime> showDatePicker({
   @required DateTime initialDate,
   @required DateTime firstDate,
   @required DateTime lastDate,
+  DateTime currentDate,
```

Comment:
Add some documentation above regarding this param.

Reply:
D'oh. I added it to `CalendarDatePicker` but not here. Fixed. Thx.